etc/thunderbird.profile

```diff
@@ -14,6 +14,12 @@ noblacklist ${HOME}/.gnupg
 # noblacklist ${HOME}/.icedove
 noblacklist ${HOME}/.thunderbird
 
+# Allow Firefox to load your profile when clicking a link in an email
```

Comment:
add info about .local e.g.:
`# Uncomment the next 4 lines or put they in your thunderbird.local to allow Firefox to load your profile when clicking a link in an email`

Reply:
Thank you, I made a new commit in order to follow your advice.